templates/buildkite-elastic.yml

```diff
@@ -170,16 +170,16 @@ Parameters:
     Description: Optional - Existing security group to associate the container instances. Creates one by default.
     Default: ""
 
-  ManagedPolicyArns:
-    Type: CommaDelimitedList
-    Description: Optional - ARNs of existing managed IAM policies to associate with the container instances.
-    Default: ""
-
   ImageId:
     Type: String
     Description: Optional - The AMI to use, otherwise uses the mapping built in
     Default: ""
 
+  ManagedPolicyARN:
+    Type: String
+    Description: Optional - ARN of an existing managed IAM policies to associate with instances
```

Comment:
That's basically correct, except they wouldn't have had to create their own policy for ECR, they just would have had to reference one of the managed AWS policies, passed in via the previous ManagedPolicyArns parameter.